admin/database/postgres/migrations/0021.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE bookmarks RENAME COLUMN dashboard_name TO resource_name;
+ALTER TABLE bookmarks ADD COLUMN resource_kind TEXT DEFAULT 'MetricsView';
```

Comment:
Should this be `rill.runtime.v1.MetricsView`?

Reply:
Yes!